### DISCUSSION TITLE
Labels not added for issue forms in shared .github repository

### DISCUSSION THREAD

**Discussion by stianst** [community/community#10488] [Projects and Issues]
With issue forms in the `.github` repository in the organisation the labels are not added to the issues, even if the labels already exists in the repository.

An example issue form we are using can be found at https://github.com/keycloak/.github/blob/main/.github/ISSUE_TEMPLATE/bug.yml

When first creating the issue the labels are added:

![image](https://user-images.githubusercontent.com/2271511/150768497-554b26b1-f0b7-4833-b431-af0b8e5a598e.png)

However, after creating the issue the labels are missing:

![image](https://user-images.githubusercontent.com/2271511/150768570-c5d078c0-afa0-487c-bd9b-1c8aac8a84d1.png)

f the user creating the issue has triage (or higher) role the labels are added as expected. Labels are also added as expected if the exact same issue forms are copied to the repository itself.

Reproduction Steps

1. Create a test issue in https://github.com/keycloak/keycloak-web
2. Observe labels are added in the issue form
3. Submit the issue and observe labels are missing

The above repository doesn't have issue forms directly added to it, but are getting the issue forms from https://github.com/keycloak/.github/tree/main/.github/ISSUE_TEMPLATE.

Also, observe that the same issue forms are working in https://github.com/keycloak/keycloak, when the issue form is copied to https://github.com/keycloak/keycloak/tree/main/.github/ISSUE_TEMPLATE (copies from the .github repository).

**Comment by shmeadyy** [comment]
Hey there @stianst! I'm a PM for GitHub doing some investigating into issue forms and I came across this feedback and even [tested it for myself in your repo](https://github.com/keycloak/keycloak-web/issues/266) - I see what you mean! I copied the exact same issue template into my own repo and was able to have the labels [appropriately applied](https://github.com/shmeadyy/furry-meme/issues/67). 

I noticed that you don't have an ISSUE_TEMPLATE folder in your .github folder in the repo? If you add that and move this .yml file to that folder, does it work then?

**Reply by stianst** [reply]
Yeah, if I copy all the forms to the repositories directly that works fine, but since we have many repos and want to keep the issue forms shared/same it would be much better if we could use the shared issue forms from the .github repository, which seems to work perfectly expect for this labelling issue.

**Comment by gerhard** [comment]
Hi, we are experiencing a similar issue with https://github.com/dagger/.github

Labels are automatically added if one of the https://github.com/dagger org users creates new issues using the issue form:
<img width="1440" alt="Screenshot 2023-05-18 at 12 15 09" src="https://github.com/community/community/assets/3342/b1902b8e-e1e8-4492-9165-7f2f0480b2d4">

Labels are **not** added if a non-org user creates them:
<img width="1440" alt="Screenshot 2023-05-18 at 12 16 24" src="https://github.com/community/community/assets/3342/8a9ce968-ffe6-4d87-89a2-cc90c446d415">

Would it be possible to have this bug fixed?

---
cc @mircubed
